Three_parameters.py

```diff
@@ -0,0 +1,22 @@
+from pprint import pprint
```

Comment:
what is that?

Reply:
this is pretty print function, to print the issues beautifully and easy to understand